### PR TITLE
Add Katello 4.7 api docs

### PR DESCRIPTION
**Pull request by chris1984**
@ekohl 

I noticed other katello versions are missing, do you want me to make pr's to add them?

### CODE REVIEW

**Comment by chris1984** [comment]
@ekohl does this need anything else or should we do another pull before GA?

File: index.html [hunk 1]
```diff
@@ -30,6 +30,7 @@ <h2>Foreman core API</h2>
 	<h2>Katello API</h2>
 	<ul>
 		<li><a href="./katello/latest/apidoc/v2.html">latest</a></li>
+		<li><a href="./katello/4.4/apidoc/v2.html">4.7</a></li>
```
Comment (ekohl) [inline index.html]:
```suggestion
		<li><a href="./katello/4.7/apidoc/v2.html">4.7</a></li>
```

**Review by ekohl — COMMENTED** [review]
Sorry, I forgot about this. One small note.

I think adding other Katello versions is actually nice for users, but it shouldn't block this PR.

**Comment by chris1984** [comment]
@ekohl committed your suggestion, can squash it when you merge.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**